lib/templateActions.js

```diff
@@ -73,7 +73,7 @@ module.exports = function (seneca, options) {
         }
 
         template = templateinstance
-        callback(null, template)
+        callback(null, {})
```

Comment:
Is this the right fix?
Should the line above perhaps read something like this instead:
`template = _.isObject(templateinstance) || _.isArray(templateinstance) ? templateinstance : {};`

cc/ @rjrodger